spec/fixtures/pages/pdf-in-nested-iframe.html

```diff
@@ -0,0 +1,7 @@
+<html>
+<body>
+<iframe id='outer-frame' src="./pdf-in-iframe.html"/>
+</body>
+</html>
+
+
```

Comment:
Empty lines are not needed here. In this file and in the html file above.
We should probably have a linter doing this kind of checks =/